src/pivot-table/components/icons/Minus.tsx

```diff
@@ -3,20 +3,21 @@ import React from "react";
 import { PLUS_MINUS_ICON_SIZE } from "../../constants";
 
 interface IconProps {
-  onClick: ((e: React.SyntheticEvent<Element, Event>) => void) | undefined;
+  onClick?: ((e: React.SyntheticEvent<Element, Event>) => void) | undefined;
   color: string;
-  testid: string;
   opacity: number;
 }
 
-const MinusIcon = ({ color, opacity, testid, onClick }: IconProps): JSX.Element => (
+export const testIdCollapseIcon = "collapse-icon";
+
+const MinusIcon = ({ color, opacity, onClick }: IconProps): JSX.Element => (
```

Comment:
Maybe, it would add more tenary operators for the consuming component. But turn two almost identical components into one.

I thinkl from a "DRY" perspective it would make a lot of sense if these icon components where expected to change a lot